airbyte-integrations/connectors/source-mysql/src/test-integration/java/io/airbyte/integrations/source/mysql/MySqlSourceComprehensiveTest.java

```diff
@@ -234,6 +234,14 @@ protected void initTests() {
             // .addInsertValues("'0000-00-00 00:00:00'")
             .build());
 
+    // addDataTypeTestData(
+    // TestDataHolder.builder()
+    // .sourceType("datetime")
+    // .airbyteType(JsonSchemaPrimitive.STRING)
+    // .addInsertValues("'2013-09-05T10:10:02'")
+    // .addExpectedValues("2013-09-05T10:10:02")
+    // .build());
```

Comment:
what's the reason for commenting this out? Is it because of the timezone adjustment you mention in the other file?

Reply:
Yes. Once we fix the implementation, we should uncomment this

Reply:
this is unrelated to the CDC change right? it's just something you discovered while working on this?

Reply:
Yup